src/Core/Commander/ManagerCommands.js

```diff
@@ -40,6 +40,9 @@ define('Core/Commander/ManagerCommands', [
             this.providerMap = {};
             this.history = null;
             this.eventsManager = new EventsManager();
+            this.currentCommandsCount = 0;
+            this.maxConcurrentCommands = 16;
```

Comment:
I'm not sure that hardcoding a command count is a good idea.
I believe the purpose is two-fold:
* limit the number of parallel network fetches (btw, 16 is too much. On Chrome lots of requests are in 'queued' state, see https://developers.google.com/web/tools/chrome-devtools/profile/network-performance/understanding-resource-timing )
* keep a high enough framerate

For the first point maybe we need a global network fetch limit (16?) + a per-host limit (Chrome documentation suggests 6).
For the second case we need a update duration limit (50 ms if we want to keep 20fps for instance).
Last but not least: some commands are not doing network job (it'll be more visible once #149 is merged).